Windows seems to be failing because:

```
======================================================================
FAIL: test_conda_build (llvmlite.tests.test_binding.TestBuild)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Miniconda\envs\test\conda-bld\llvmlite_1777445991762\_test_env\lib\site-packages\llvmlite\tests\test_binding.py", line 3147, in test_conda_build
    self.assertEqual(info['llvm_assertions_state'], "on")
AssertionError: 'off' != 'on'
- off
+ on
```

Looking at the build job, I believe it seems to be getting the wrong `llvmdev`:

```
    defaults/win-64::llvmdev-20.1.8-h48dad37_1
```

--- a/.github/workflows/llvmlite_conda_builder.yml
+++ b/.github/workflows/llvmlite_conda_builder.yml
@@ -120,8 +120,7 @@ jobs:
           else
             extra_args=()
           fi
-          conda-build --debug -c "${LLVMDEV_CHANNEL}" "${extra_args[@]}" -c defaults --python=${{ matrix.python-version }} conda-recipes/llvmlite --output-folder="${CONDA_CHANNEL_DIR}" --no-test
-
+          conda-build --debug --channel-priority strict -c "${LLVMDEV_CHANNEL}" "${extra_args[@]}" -c defaults --python=${{ matrix.python-version }} conda-recipes/llvmlite --output-folder="${CONDA_CHANNEL_DIR}" --no-test
       - name: Upload llvmlite conda package
         uses: actions/upload-artifact@043fb46d1a93c77aae656e7c1c64a875d1fc6a0a # v7.0.1
         with:

--- a/.github/workflows/llvmlite_conda_builder.yml
+++ b/.github/workflows/llvmlite_conda_builder.yml
@@ -120,7 +120,8 @@ jobs:
           else
             extra_args=()
           fi
-          conda-build --debug --channel-priority strict -c "${LLVMDEV_CHANNEL}" "${extra_args[@]}" -c defaults --python=${{ matrix.python-version }} conda-recipes/llvmlite --output-folder="${CONDA_CHANNEL_DIR}" --no-test
+          conda config --set channel_priority strict
+          conda-build --debug -c "${LLVMDEV_CHANNEL}" "${extra_args[@]}" -c defaults --python=${{ matrix.python-version }} conda-recipes/llvmlite --output-folder="${CONDA_CHANNEL_DIR}" --no-test
       - name: Upload llvmlite conda package
         uses: actions/upload-artifact@043fb46d1a93c77aae656e7c1c64a875d1fc6a0a # v7.0.1
         with:

